lib/cmd/fh3/app/clean-clone.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+var millicore = require("../../../utils/millicore.js");
+var fhreq = require("../../../utils/request");
+
+module.exports = cleanClone;
+
+cleanClone.usage = "fhc app clean-clone <appId>";
+cleanClone.desc = "Performs a clean clone of a git repo within the studio to resolve potential git issuse";
```

Comment:
Please pass this string to i18n._() to enable localization. please see https://github.com/feedhenry/fh-fhc/commit/c5b56cc46392b199315b496fa7c5a02f43d90c99 for more details.